src/exabgp/reactor/loop.py

```diff
@@ -172,12 +172,16 @@ def neighbor(self, peer_name):
             return
         return peer.neighbor
 
-    def neighbor_name(self, peer_name):
+    def neighbor_name(self, peer_name, json=False):
         peer = self._peers.get(peer_name, None)
         if not peer:
             log.critical('could not find referenced peer', 'reactor')
             return ""
-        return peer.neighbor.name()
+        if json:
```

Comment:
The if statement is not required, `return peer.neighbor.name(json)` does the same thing AFAICS.